homeassistant/components/notify/nfandroidtv.py

```diff
@@ -0,0 +1,192 @@
+"""
+Notifications for Android TV notification service.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/notify.nfandroidtv/
+"""
+import os
+import logging
+import requests
+import voluptuous as vol
+
+from homeassistant.components.notify import (ATTR_TITLE,
+                                             ATTR_TITLE_DEFAULT,
+                                             ATTR_DATA,
+                                             BaseNotificationService,
+                                             PLATFORM_SCHEMA)
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_IP = 'ip'
+CONF_DURATION = 'duration'
+CONF_POSITION = 'position'
+CONF_TRANSPARENCY = 'transparency'
+CONF_COLOR = 'color'
+CONF_INTERRUPT = 'interrupt'
+CONF_TIMEOUT = 'timeout'
+
+DEFAULT_DURATION = 5
+DEFAULT_POSITION = 'bottom-right'
+DEFAULT_TRANSPARENCY = 'default'
+DEFAULT_COLOR = 'grey'
+DEFAULT_INTERRUPT = False
+DEFAULT_TIMEOUT = 5
+
+ATTR_DURATION = 'duration'
+ATTR_POSITION = 'position'
+ATTR_TRANSPARENCY = 'transparency'
+ATTR_COLOR = 'color'
+ATTR_BKGCOLOR = 'bkgcolor'
+ATTR_INTERRUPT = 'interrupt'
+
+POSITIONS = {
+    "bottom-right": 0,
+    "bottom-left": 1,
+    "top-right": 2,
+    "top-left": 3,
+    "center": 4,
+}
+
+TRANSPARENCIES = {
+    "default": 0,
+    "0%": 1,
+    "25%": 2,
+    "50%": 3,
+    "75%": 4,
+    "100%": 5,
+}
+
+COLORS = {
+    "grey": "#607d8b",
+    "black": "#000000",
+    "indigo": "#303F9F",
+    "green": "#4CAF50",
+    "red": "#F44336",
+    "cyan": "#00BCD4",
+    "teal": "#009688",
+    "amber": "#FFC107",
+    "pink": "#E91E63",
+}
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_IP): cv.string,
+    vol.Optional(CONF_DURATION, default=DEFAULT_DURATION): vol.Coerce(int),
+    vol.Optional(CONF_POSITION, default=DEFAULT_POSITION):
+        vol.In(POSITIONS.keys()),
+    vol.Optional(CONF_TRANSPARENCY, default=DEFAULT_TRANSPARENCY):
+        vol.In(TRANSPARENCIES.keys()),
+    vol.Optional(CONF_COLOR, default=DEFAULT_COLOR):
+        vol.In(COLORS.keys()),
+    vol.Optional(CONF_COLOR, default=DEFAULT_COLOR): cv.string,
+    vol.Optional(CONF_TIMEOUT, default=DEFAULT_TIMEOUT): vol.Coerce(int),
+    vol.Optional(CONF_INTERRUPT, default=DEFAULT_INTERRUPT): cv.boolean,
+})
+
+
+# pylint: disable=unused-argument
+def get_service(hass, config):
+    """Get the Notifications for Android TV notification service."""
+    remoteip = config.get(CONF_IP)
+    duration = config.get(CONF_DURATION)
+    position = config.get(CONF_POSITION)
+    transparency = config.get(CONF_TRANSPARENCY)
+    color = config.get(CONF_COLOR)
+    interrupt = config.get(CONF_INTERRUPT)
+    timeout = config.get(CONF_TIMEOUT)
+
+    return NFAndroidTVNotificationService(remoteip,
```

Comment:
That is a correct assumption. I meant that you check if the given IP actually contains a Notification for Android TV service